client/src/modules/prices/prices.js

```diff
@@ -48,6 +48,12 @@ function PriceListController(
       headerCellFilter : 'translate',
       cellTemplate : `/modules/prices/templates/itemsNumber.cell.html`,
     },
+    {
+      field : 'subcribedGroupsNumber',
+      displayName : 'PATIENT_GROUP.PATIENT_GROUP',
+      headerCellFilter : 'translate',
+      cellTemplate : `/modules/prices/templates/subscribedGroups.cell.html`,
```

Comment:
Why did you need to make a cell for this?

Reply:
Hum..., I was trying to add this subscribers number where possible to end with this problem

Reply:
But do you need a template?  What happens if you remove the cell template?